sema/bench/__main__.py

```
@@ -0,0 +1,108 @@
+        watch_config_file=args.watch,
+        fail_fast=args.fail_fast,
+    )
+
+
+def _main(*args_list: str) -> bool:
+    args = get_arg_parser().parse_args(args_list)
+
+    try:
+        sembench = make_service(args)
+        sembench.run()
+    except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)